### DISCUSSION TITLE
ISO Command not enabled?

### DISCUSSION THREAD

**Discussion by jamaya77** [ipxe/ipxe#490] [Q&A] [answered]
I'm trying different ways to deploy Parted Magic from iPXE. I've successfully deployed it Parted Magic, based on their [documentation](https://partedmagic.com/pxe/), but I'm in need to deploy the whole Parted Magic iso.

I've tried to run a script using the ISO command, but it looks looks like it's not working or not enabled? If so, how do I enable the iso command? See attached snapshot. Thank you.

#!ipxe

initrd http://192.168.140.21/iso/pmagic/pmagic_2021_09_15.iso
chain http://192.168.140.21/iso/pmagic/memdisk iso

boot

![isoipxe1](https://user-images.githubusercontent.com/80473349/136310955-4851e32d-6f52-4c09-bbd3-19dedb609ffb.png)

**Comment by NiKiZe** [comment, marked as answer]
There is no iso command, what you have done here is to try and start the memdisk binary, are you running this in efi?

You are unlikely to be able to get this method to work, if you where to use sanboot instead you will probably be able to get first part of the boot process running. That is running bootloader, load kernel and initrd. But once the kernel start it will not be able to find other files that the init scripts might need, since the iso simply isn't available after the kernel starts.

You can read more about this in the FAQ: https://forum.ipxe.org/showthread.php?tid=5948 see the part about memdisk.

The error message and URL: http://ipxe.org/2e008081 also gives the information about EFI environment and legacy binaries.

**Reply by jamaya77** [reply]
Thank you, NikiZe, sanboot worked for me.
